gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -351,6 +244,202 @@ protected GoogleCloudStorageImpl() {
     this.storageOptions = GoogleCloudStorageOptions.builder().setAppName("test-app").build();
   }
 
+  // A function to encode metadata map values
+  private static String encodeMetadataValues(byte[] bytes) {
```

Comment:
This seems like not strictly code move - it will complicate merging of gRPC code into other branches, please revert.

Reply:
Ah, looks like I accidentally formatted the file with IntelliJ, which is apparently rather opinionated about where to put functions. Reverted all the ordering changes.